components/cam/src/physics/rrtmgp/radiation_utils.F90

```diff
@@ -285,9 +285,10 @@ end subroutine clip_values_2d
 
    subroutine handle_error(error_message, stop_on_error)
       use cam_abortutils, only: endrun
+      use cam_logfile, only: iulog
       character(len=*), intent(in) :: error_message
       logical, intent(in), optional :: stop_on_error
-      logical :: stop_on_error_local = .true.
+      logical :: stop_on_error_local = .false.
```

Comment:
This change won't actually do anything, because `stop_on_error_local` is set explicitly below in all cases. Regardless, I think this is a bit dangerous in hindsight, because setting a value on declaration like this I think gives it an implicit "save", so it would be possible for this to cause issues with threading if it weren't set explicitly below.

Reply:
duh you're right. I totally misread the logic.  I'll just feed the routine the optional argument when it checks the optics values.

Reply:
Okay that sounds good. I do think we want to limit the relaxation of the error handling to just the spot where we were running into problems, rather than allow the model to keep running in all cases an error is returned from RRTMGP. So that sounds like a good plan, just pass `stop_on_error=.false.` to the validation of the aerosol optics where it was failing occasionally.

Reply:
I agree.  Fixed this and tested it.  Of course, now with this version of the code the error is not happening at all in a one month run (which is my my original false implementation didn't bail out).  Will continue to monitor the frequency of this error.  If it seems to go away completely in upcoming SCREAM development simulations then we can remove the optional flag someday.